styleguide/components/01-mixins/themify/_themify.scss

```diff
@@ -1,152 +1,91 @@
-  $hamburger-menu-link-background-active: map-get($colors, 'hamburger-menu-link-background-active') !global;
-  $hamburger-menu-link-background-hover: map-get($colors, 'hamburger-menu-link-background-hover') !global;
-
-  $hamburger-menu-link-color: map-get($colors, 'hamburger-menu-link-color') !global;
-  $hamburger-menu-link-color-active: map-get($colors, 'hamburger-menu-link-color-active') !global;
-  $hamburger-menu-link-color-hover: map-get($colors, 'hamburger-menu-link-color-hover') !global;
-
-  $island-border: map-get($colors, 'island-border') !global;
+// @param $default string
+//   The default value for the themified property.
+//
+// @param $exception string
+//   The exception value for the themified property.
+//   This is the exception optionally used in subthemes.
+//
+@function set-themified-property($colors, $default, $exception) {
+  @if map-has-key($colors, $exception) {
+    @return map-get($colors, $exception);
+  }
+  @else {
+    @return map-get($colors, $default);
+  }
 }
 
 //
-// Themify mixin
+// Theme mixin to generate all themes for components.
+// This generates the necessary theme CSS for all defined themes based on the
+// $themes map defined in _colors.scss
 //
 // @access public
 //
-// @param {Map} $themes [$themes] - Map of themes to use
+// @param $property string
+//   The property to themify.
+//
+// @param $default string
+//   The default value for the themified property.
 //
-@mixin themify($themes: $themes) {
+// @param $exception string
+//   The exception value for the themified property.
+//   This is the exception optionally used in subthemes.
+//
+@mixin theme($property: null, $default: null, $exception: null) {
   @each $theme, $colors in $themes {
-    @include themify-map($colors);
-
     // Apply the default section as default color scheme if there is no section
     // defined in the DOM.
     @if $theme == "default" {
-      @content;
+      #{$property}: set-themified-property($colors, $default, $exception);
     }
 
-    .cs--#{$theme} &,
-    *[class*="cs--"] .cs--#{$theme} &,
-    *[class*="cs--"] %cs--#{$theme} & {
-      @content;
+    .cs--#{$theme} & {
+      #{$property}: set-themified-property($colors, $default, $exception);
     }
   }
 }
 
 //
-// Merge {Map} $themes map if {Map} $overwrite-themes exist.
+// Theme mixin to generate all themes for components.
+// This generates the necessary theme CSS for all defined themes based on the
+// $themes map defined in _colors.scss
 //
-$themes: map-merge($themes, $overwrite-themes);
+// @access public
+//
+// @param $property string
+//   The property to themify.
+//
+// @param $default string
+//   The default value for the themified property.
+//
+// @param $exception string
+//   The exception value for the themified property.
+//   This is the exception optionally used in subthemes.
+//
+@mixin theme($property: null, $default: null, $exception: null) {
```

Comment:
You already defined this mixin at line https://github.com/StadGent/drupal_theme_gent-base/pull/35/files/f1887c441be4b6d44d82f51b00ff97b58376c120..1fe5660111d7fe50417c46d1ec929f19b1a7662e#diff-dcef1d3666b042d1343a8b650c7a1827R47

Reply:
Oh wow totally missed that. Removed it.